### PR TITLE
Split metrics to multiple http sources

### PR DESCRIPTION
**Pull request by samjsong**
We want to split metrics to multiple HTTP sources to prevent blacklisting. Synced with Frank about the more common jobs we'd want to separate out, which included:
```
apiserver
cadvisor
kubelet
kube-controller-manager
kube-scheduler
kube-state*
node-exporter
prometheus-operator-*
```
We still include a catchall http source for those metrics not belonging to any of the above jobs.

@frankreno @bin3377 @lei-sumo 

FYI - @rvmiller89 @ggarg2906sumo

### CODE REVIEW

File: deploy/kubernetes/fluentd-sumologic.yaml [hunk 1]
```diff
@@ -78,8 +86,65 @@ data:
     </match>
     <filter prometheus.datapoint>
       @type carbon_v2
-    </filter>    
+    </filter>
     <match prometheus.datapoint>
+      @type rewrite_tag_filter
```
Comment (bin3377) [inline deploy/kubernetes/fluentd-sumologic.yaml]: Is that possible to move this filter before transform to `carbon_v2`?
rule may be something like `key=job`, `pattern=^(.*)$`.
The reason is we may move to Prometheus format later so we don't want it impacted

Comment (samjsong) [reply deploy/kubernetes/fluentd-sumologic.yaml]: That makes sense. Let me try this and get back to you.

Comment (samjsong) [reply deploy/kubernetes/fluentd-sumologic.yaml]: It seems to have worked! resolving

File: deploy/kubernetes/fluentd-sumologic.yaml [hunk 2]
```diff
@@ -76,10 +84,67 @@ data:
       @type datapoint
       tag "prometheus.datapoint"
     </match>
-    <filter prometheus.datapoint>
-      @type carbon_v2
-    </filter>    
     <match prometheus.datapoint>
+      @type rewrite_tag_filter
```
Comment (lei-sumo) [inline deploy/kubernetes/fluentd-sumologic.yaml]: What does this part do?

Comment (samjsong) [reply deploy/kubernetes/fluentd-sumologic.yaml]: The `rewrite_tag_filter` output plugin allows you to rewrite the tag for rerouting. In this case the plugin looks for the job and appends it to the already existing tag, which becomes `prometheus.datapoint.<JOB>`

**Comment by samjsong** [comment]
cc @ggarg2906sumo this PR has been merged, so now you will need multiple (nine) http sources on the sumo side

**Pull request merged**